.github/workflows/ci.yml

```diff
@@ -114,7 +115,43 @@ jobs:
       # Ensure all downstream packlet versions are consitent with the base packlet version
       - name: Ensure all package versions are consistent
         run: rush ensure-consistent-versions
-      # Tests
+      # Upload azure-communication api.md files for easy access
+      - name: Upload communication-react api files
+        uses: actions/upload-artifact@v3
+        with:
+          name: communication-react.${{ matrix.flavor }}.api.json
+          path: |
+            packages/communication-react/temp/communication-react.api.json
+
+  jest-test-coverage:
+    needs: [get_matrix, build_packages]
```

Comment:
As this job has its own build step, we don't need to wait for `build_packages` job to be completed

Reply:
We don't need it, this is true. But there's no point running it unless the build step completes successfully

Reply:
Build job also includes linting and treeshaking (~2 minutes) that can be skipped for tests. Tests will finish or almost will finish their run during that time.
Or are we going to move those too out of the build job?

Reply:
These aren't the critical path, so it doesn't really matter how long these tests take to run.
Yes, building also does linting too.
I tried moving that out to a pre-build job, but it doesn't make the process quicker.